flutter/lib/src/navigation/time_to_display_transaction_handler.dart

```diff
@@ -0,0 +1,80 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'package:meta/meta.dart';
+import '../../sentry_flutter.dart';
+import '../native/sentry_native.dart';
+
+@internal
+class TimeToDisplayTransactionHandler {
+  final Hub? _hub;
+  final bool? _enableAutoTransactions;
+  final Duration? _autoFinishAfter;
+  final SentryNative? _native;
+
+  TimeToDisplayTransactionHandler({
+    required Hub? hub,
+    required bool? enableAutoTransactions,
+    required Duration? autoFinishAfter,
+  })  : _hub = hub ?? HubAdapter(),
+        _enableAutoTransactions = enableAutoTransactions,
+        _autoFinishAfter = autoFinishAfter,
+        _native = SentryFlutter.native;
+
+  Future<ISentrySpan?> startTransaction(String? routeName, Object? arguments,
```

Comment:
we may move this method inside `TimeToDisplayTracker` and remove this class, as it's only used there